h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2oinit.py

```diff
@@ -75,10 +78,56 @@ def h2oinitname():
     except H2OConnectionError as e:
         print("error message type is {0} and the error message is {1}\n".format(e.__class__.__name__, e.args[0]))
         assert_is_type(e, H2OConnectionError)
+        h2o.cluster().shutdown()
+
+
+def h2oinit_default_log_dir():
+    tmpdir = tempfile.mkdtemp()
+    try:
+        h2o.init(strict_version_check=False, name="default_log", ice_root=tmpdir)
+    except H2OConnectionError as e:  # some errors are okay like version mismatch
+        print("error message type is {0} and the error message is {1}\n".format(e.__class__.__name__, e.args[0]))
```

Comment:
I'd make these error messages more informative. Mention error during startup with custom log dir. My only reason is these are customer facing. The more informative our errors are, the more are users able to solve themselves.